<launch>

<rosparam file="$(find inmoov_bringup)/config/config.yaml"/>

<group ns="servobus/00" >
  <node pkg="rosserial_python" type="serial_node.py" name="bus" respawn="true">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="115200"/>
  </node>
</group>

<group ns="servobus/01" >
  <node pkg="rosserial_python" type="serial_node.py" name="bus" respawn="true">
      <param name="port" value="/dev/ttyACM1"/>
    <param name="baud" value="115200"/>
  </node>
</group>

<group ns="servobus/02" >
  <node pkg="rosserial_python" type="serial_node.py" name="bus" respawn="true">
    <param name="port" value="/dev/ttyACM2"/>
    <param name="baud" value="115200"/>
  </node>
</group>

<node pkg="inmoov_bringup" type="generate_export_yaml.py" name="generate_export_yaml" respawn="false"/>

</launch>